src/main/scala/WeaviateDataWriter.scala

```diff
@@ -42,7 +42,7 @@ case class WeaviateDataWriter(weaviateOptions: WeaviateOptions, schema: StructTy
       val errors = objectsWithError.map(obj => s"${obj.getId}: ${obj.getResult.getErrors.toString}")
       val successIDs = objectsWithSuccess.map(_.getId).toList
       logWarning(s"Successfully imported ${successIDs}. " +
-        s"Retrying objects with an error. Following objects in the batch upload had an error: ${errors}")
+        s"Retrying objects with an error. Following objects in the batch upload had an error: ${errors.mkString("Array(", ", ", ")")}")
```

Comment:
this looks weird, but I assume this makes the errors look like this?
```
Array(
  error1, error2, error3
)
```
the multiple `,` is throwing me off but looks correct, `prefix separator, suffix`